### DISCUSSION TITLE
syntax question for email search

### DISCUSSION THREAD

**Discussion by paulobreim** [sepinf-inc/IPED#1764] [Q&A]
I'm trying to do a search for emails sent from one person to another, and I have some questions.
1 - Accessing the rectangle below, on the left (metadata), I select the Communication Group and also selecting Communication:From in the properties and placing the e-mail I want to search for in the values field, I manage to obtain the correct result (by the way, I don't know what it is the first properties field, I left it blank).

But since I also want to select the recipient, I first tried to do the search in the horizontal bar that we normally use, but I can't get the syntax right.
When I put:
Communication:From:user@domain.com.br returns nothing.
I tried too
communication\:from:user@domain.com.br
but I was not successful.
Can anyone tell me the correct syntax?

tks

**Comment by wladimirleite** [comment]
A tip that may help, just type "from" (without quotes) and the press TAB (one or more times) until you find the desired property. Then insert the value with enclosing quotes (something like "user@domain.com"). @ and the dot won't be searched, but they will work as a separator.

**Comment by lfcnassif** [comment]
Here it is:
`Communication\:From:"user1@domain.com" AND Message\-Recipient\-Address:"user2@domain.com"`

`Message\-Recipient\-Address` looks into TO, CC and BCC.

If you want to search in both directions:
`(Communication\:From:"user1@domain.com" AND Message\-Recipient\-Address:"user2@domain.com") OR (Communication\:From:"user2@domain.com" AND Message\-Recipient\-Address:"user1@domain.com")`

**Reply by paulobreim** [reply]
Excellent @lfcnassif , thank you very much.
I've tested several variants and they all work, including things like *@gmail.com, for example.

paulo
